data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
+				`	(Leave Caesar here.)`
+
+			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`
+			`	As you turn away, Caesar begins to whine, but soon quiets down when you don't come back to pick him up. Looking back, you notice him slink off into the shadow of a perfectly manicured shrub.`
+				decline
+
+			label takehim
+			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`
+			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`
+				accept
+
+	on complete
+		payment 1000
+		conversation
+			branch "transported pookie"
+				has "Pookie, Part 2: done"
+				not `outfit: "Puny"`
+			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads "Mitchell Ranch."`
+				goto ranch
+
+			label "transported pookie"
+			`Caesar has been very obedient the whole trip, unlike some other certain dogs you've transported in the past. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
+
+			label ranch
+			`	It's another forty-five minutes of driving down long winding roads, past various outbuildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`
+			`	The butler arrives and looks you over unhappily. "The lady of the manor is not expecting... guests. Perhaps, you've misread your appointment, or more likely, arrived at the wrong estate."`
+			choice
+				`	(Show the butler Caesar, and ask to see his owner.)`
+
+			`	The butler's frown deepens, and then sighs. "Card, please." He holds out a silver tray and awaits your calling card.`
+			choice
+				`	(Explain you don't have a calling card.)`
```

Comment:
Who knows how many years its been since then. Plus maybe you were a big fan of space downton abby? Or it's a more ubiquitous part of their culture than it is ours.

Reply:
I think it's part of the intro, so I assume you quit your job right before the start date.

Reply:
Sorry, I was editing my response above when you replied. So they may not mesh anymore.